### PR TITLE
feat: add icons to draw attention to date config

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Add some icons to draw attention to date config

JIRA: [AU-1432](https://2u-internal.atlassian.net/browse/AU-1432)

**What changed?**

- Added fa star to the schedule tab
- Added a NEW badge next to the config radio selct

Screenshot 2023-09-01 at 11 15 46 AM

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/xblock/static/sass/oa/utilities/_developer.scss [hunk 1]
```diff
@@ -1077,6 +1077,18 @@
         }
       }
     }
+
+    // TODO: Remove this in AU-1433
```
Comment (mattcarter) [inline openassessment/xblock/static/sass/oa/utilities/_developer.scss:1081]: Nit: Can you explain here the goal of the referenced ticket ie: "Decide feature is no longer new and remove tagging"

**Review by schenedx — APPROVED** [review]
👍
